File: proxy/tests/script_engine/notify.js
/// <reference path="../../script_libs/js/index.d.ts" />
/** @type {Extension} */
const notify = {
  request() {
    globalThis.notify(1, "hi")
  },
  response() {
    globalThis.notify(2, "there")
  }
}
globalThis.extensions = [notify];

File: proxy/tests/script_engine/notify.lua
pcall(require, "../../script_libs/lua/roxy.lua")
---@type Extension
local notify = {
	request = function()
		Roxy.notify(1, "hi")
	end,

	response = function()
		Roxy.notify(2, "there")
	end,
}
Extensions = { notify }
